Sources/CryptomatorCloudAccess/WebDAV/WebDAVProvider.swift

```diff
@@ -7,14 +7,15 @@
 //
 
 import Foundation
+import GRDB
```

Comment:
```suggestion
```

Unused import, right?

Reply:
No, we need GRDB to initialize the `DatabaseQueue` (see line 43):
```swift
self.directoryContentCache = try DirectoryContentDBCache(dbWriter: DatabaseQueue(path: dbURL.path), maxPageSize: maxPageSize)
```